storage/migrations/22_evm_nfts_grant.up.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+
+-- Grant others read-only use.
+GRANT SELECT ON ALL TABLES IN SCHEMA chain TO PUBLIC;
+GRANT EXECUTE ON ALL FUNCTIONS IN SCHEMA chain TO PUBLIC;
```

Comment:
nit: If `21_evm_nfts.up.sql` hasn't been applied yet to prod, I'd also be fine with merging this with that migration. This way is also completely fine too.

Reply:
it was in 0.1.16 :grimacing: